packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -2106,7 +2136,7 @@ QUnit.module('Options', {
             });
 
             QUnit.test('It should be possible to deselect already selected value by click', function(assert) {
-                const $cell = $(getCurrentViewInstance(this.calendar).$element().find('*[data-value="2023/01/15"]'));
+                const $cell = $(getCurrentViewInstance(this.calendar).$element().find('*[data-value="2023/02/01"]'));
```

Comment:
Same here. We should be able to deselect any date, why then it's so important to update here?